### ISSUE TITLE
Error when using a string with the content "</script>"

### ISSUE THREAD

**Issue by Rickyp** [OfficeDev/script-lab#595]
Host: Excel online, using Chrome
There is a parsing/compiling error somewhere that happens when the user code contains a string with the content "</script>". For example:

// User code
$("#run").click(() => tryCatch(run));

async function run() {
  console.log("</script>");
}

/** Default helper for invoking an action and handling errors. */
async function tryCatch(callback) {
  try {
    await callback();
  } catch (error) {
    OfficeHelpers.UI.notify(error);
    OfficeHelpers.Utilities.log(error);
  }
}

EXPECTED:
It console logs "</script>"

ACTUAL:
![image](https://user-images.githubusercontent.com/7102621/52000580-5c01bc00-2472-11e9-9a4c-83491daf7701.png)

**Comment by Zlatkovsky** [comment]
Good catch, super-interesting.  I thought we were escaping the string inside the script, but looks like we're not.  I can take this one.

**Comment by Zlatkovsky** [comment]
BTW, if you're blocked, I'm pretty sure that if you switch to Script Lab 2017 ([see instructions](https://github.com/OfficeDev/script-lab/blob/master/README.md#2017)), this should work for you.

**Comment by ElizabethSamuel-MSFT** [comment]
Closing all issues since this repo is being archived and no longer maintained.
